### PR TITLE
Issue #4 Solved

### PR DESCRIPTION
**Pull request by shudhanshu03**
#4 
Changed the background with attractive image and used linear gradient for making it cool.

### CODE REVIEW

**Comment by Aakansha99** [comment]
Please add screenshot for the same

**Comment by shudhanshu03** [comment]
Just a sec @Aakansha99 . Uploading it....

**Comment by shudhanshu03** [comment]
Home Page 
Screenshot (80)


Compare Page 
Screenshot (81)

File: index.html [hunk 1]
```diff
@@ -9,8 +9,8 @@
             Home
         </title>
     </head>
-    
-    <body class="home">
+
+    <body class="home background_Image">
```
Comment (frantic32211) [inline index.html:13]: check the class name with the CSS file.

Comment (shudhanshu03) [reply index.html:13]: > check the class name with the CSS file.

Its correct ✔️  and its working fine too✌🏼

**Pull request closed without merging**